src/main/webapp/react-hooks/use-get-permissions.tsx

```diff
@@ -19,12 +26,12 @@ const user = gql`
   }
 `
 
-const usePermissions = (user: User) => {
-  return (attributes: any) => {
+const usePermissions = (user: User = {}) => {
+  return (attributes: any = {}) => {
     const securityAttributes = getSecurityAttributesFromMetacard(attributes)
     return getPermissions(
-      user.email,
-      user.roles,
+      user.email || '',
+      user.roles || [],
```

Comment:
with these changes do you still need, this change? https://github.com/connexta/revelio/pull/333/files#diff-8e61c7dfd17cef5f94afbba276fdbc9eR56

Reply:
An undefined value still may be returned if the user query is still loading https://github.com/connexta/revelio/pull/333/files/e8340ec91a66c723502d36849e1ebd797cb8eabd#diff-fc3b9df5837400effc9fe86ea4d16242R44